src/wrap/mod.rs

```diff
@@ -105,9 +131,22 @@ mod tests {
             stream,
         ));
 
+        let wrapped = textwrap::Wrapper::new(WrapConfig::default().max_line_length)
+            .initial_indent(" ")
```

Comment:
:+1:

Reply:
We should probably consider tracking if it's a `///` comment or a `#[doc=...` style comment, for sake on how to rewrap things